src/osemgrep/cli_login/dune

```diff
@@ -7,6 +7,9 @@
     cmdliner
     commons
 
+    semgrep_core_cli
```

Comment:
Why do you need semgrep_core_cli?
osemgrep should not depend on semgrep_core_cli because at some point we will remove it
(because there will be no need for having osemgrep and semgrep-core; we would have just osemgrep.

Reply:
for `Version.version` (cli_scan as well uses that) -- but if we don't emit the log message above with version and such, we can just get rid of that :)

Reply:
removed in [9306c04](https://github.com/returntocorp/semgrep/pull/7637/commits/9306c04c582e7d6724195fea1e0b102f0dc9335a)

Reply:
yes, just remove those